spec/models/data_analysis/local_authority_user_spec.rb

```diff
@@ -22,6 +22,11 @@
   end
 
   before do
+    # Incomplete registration
+    create :user, :confirmed, local_authority: 'LA3'
```

Comment:
Looks fine other than this factory doesn't occur in the wild. Anyone who has confirmed is first asked for their name before their LA. So we could just remove this line and things will still pass.